### DISCUSSION TITLE
JMeter Report Issue?

### DISCUSSION THREAD

**Discussion by amracel** [qualersoft/jmeter-gradle-plugin#149] [Q&A] [answered]
We've switched from an unsupported plugin to the one you've created (thank you!). But we're having an issue with the creation of reports. The 'old' plugin could create a report (jMeterTestReport), but your plugin gives me an error related to Java 17. Other searches (https://stackoverflow.com/questions/69294826/error-while-generating-report-in-apache-jmeter-5-4-1-tgz/69311637#69311637) say this issue is fixed in JMeter 5.5 (which I have). Any ideas why I'm getting it running using your plugin?

**Comment by mathze** [comment, marked as answer]
Hi @amracel,

the current version (2.4.0) of the plugin uses jmeter `5.4.1`. So to use the 5.5 version just put
```
jmeter {
  tool {
    version = "5.5"
  }
}
```
in your `build.gradle.kts`. Tested it successfully on my machine (Win11 and java Temurin-17.0.3+7 (build 17.0.3+7))

Please let me know if it work.

One thing I've noted:
I'm not quite sure what you mean by "... (**which I have**) ...", but if I understand correctly you mean you've put JMeter version 5.5 somewhere on your machine, right?
If that's true, then this will not effect the plugin as it downloads its own JMeter into the build-folder.
The idea behind this is, that people working with your project don't need to download and setup JMeter and also don't need care about the right version.

I guess I should state this in the documentation more explicit.

**Reply by amracel** [reply]
Thanks. yes, that did end up being the issue.
